build.gradle

```diff
@@ -36,38 +36,38 @@ String[] classpathLibraries = [
         'lib/bundles/com.google.guava_18.0.0.jar',
         'lib/bundles/com.nomagic.annotations_1.0.0/com.nomagic.annotation-1.0.jar',
-        'lib/bundles/com.nomagic.ci.binary_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.ci.metamodel.project_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.ci.persistence.local_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.ci.persistence_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.ci.services_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.ci_17.0.5.v20191219-1130.jar',
-        'lib/bundles/com.nomagic.magicdraw.ce_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.core.diagram_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.core.project.options_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.core_1.0.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.esi.binary.metamodel_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.esi.esiproject_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.esi.uuidmap_18.1.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.foundation_2.0.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.osgi.fragment_1.0.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.security_18.3.0.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw.uml2_2.5.1.201912191136.jar',
-        'lib/bundles/com.nomagic.magicdraw_17.0.5.201912191136.jar',
+        'lib/bundles/com.nomagic.ci.binary_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.ci.metamodel.project_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.ci.persistence.local_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.ci.persistence_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.ci.services_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.ci_17.0.5.v20200602-0758.jar',
+        'lib/bundles/com.nomagic.magicdraw.ce_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.core.diagram_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.core.project.options_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.core_1.0.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.esi.binary.metamodel_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.esi.esiproject_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.esi.uuidmap_18.1.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.foundation_2.0.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.osgi.fragment_1.0.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.security_18.3.0.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw.uml2_2.5.1.202006020804.jar',
+        'lib/bundles/com.nomagic.magicdraw_17.0.5.202006020804.jar',
```

Comment:
These version updates don't match 19.0 SP3 or SP4. Can you clarify the version these changes are dependent on?

From my digging SP4 has `com.nomagic.magicdraw_17.0.5.202003140800.jar` and SP3 has `com.nomagic.magicdraw_17.0.5.201912191136.jar`. We also need to update the public bundle dependencies to match. `./gradlew clean runJava` can be used to ensure it runs.